src/current/cockroachcloud/egress-private-endpoints.md

```diff
@@ -40,7 +40,7 @@ curl --request GET \
   --header 'Authorization: Bearer {secret_key}' | jq .account_id
 ~~~
 
-### MSK endpoints
+### MSK
```

Comment:
nit: AWS MSK

Reply:
Are there equivalent prerequisites for GCP MSK?

Reply:
There is no GCP MSK. "MSK" stands for Managed Streaming for Apache Kafka, and its a product distinct to AWS: https://aws.amazon.com/msk/

If you think its more clear to call it "MSK" instead of "AWS MSK" thats fine with me, your call